app/controllers/admin/posts_controller.rb

```diff
@@ -19,6 +20,9 @@ def edit
   def create
     @post = Post.new(post_params)
     if @post.save
+      params[:photos]['image'].each do |photo|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.